src/main/java/decodes/dbeditor/ConfigSensorEditDialog.java

```diff
@@ -154,9 +155,9 @@ public ConfigSensorEditDialog(ConfigSensor cs, boolean csNewlyAdded)
 			 && theSensor.getUsgsStatCode() == null)
 				theSensor.setUsgsStatCode(sc); 
 
-		    propertiesEditPanel = new PropertiesEditPanel(theProperties);
+		    propertiesEditPanel = PropertiesEditPanel.from(theProperties);
```

Comment:
Can we make a global fix so our indentations are standardized?

Reply:
Wow, yeah. didn't realize how out of whack that file was.